pkg/jobrunaggregator/tablescreator/table_creator.go

```diff
@@ -48,5 +48,19 @@ func (r *allJobsTableCreatorOptions) Run(ctx context.Context) error {
 		fmt.Fprintf(os.Stdout, "table already exists: %s\n", jobrunaggregatorlib.JobRunTableName)
 	}
 
+	// Create TestRunsTable
+	testRunsTable := r.ciDataSet.Table(jobrunaggregatorlib.TestRunTableName)
```

Comment:
We now have several views on top of this table.  How would you like to handle those?  Would you consider an approach that has a map of table name to schema definition and have this command create from that?

Reply:
Let me see if I follow: make `map[string]string` (first string is tableName, second string is Schema represented in Json string), modify the function to create all the tables using `range` over the map keys? -- does that sound right?

I think existing views on top of `TestRuns` table would remain intact and unaffected (because today, they don't depend on the new `TestSuite` field).

Reply:
My understanding of David's comment is more specific to views, which as far as I know, are not created by code today, only by hand. I think he was asking about what we can do to change that, probably in followup PRs.